### PR TITLE
Styled the "view all x candidates" and "show more" links and added ios "arrow-forward" icon

### PR DESCRIPTION
**Pull request by uroshima**
### What github.com/wevote/WebApp/issues does this fix?
#1961

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -28,6 +28,7 @@
     <link rel="icon" type="image/png" sizes="32x32" href="/img/global/icons/favicon-32x32.png" >
     <link rel="icon" type="image/png" sizes="16x16" href="/img/global/icons/favicon-16x16.png" >
     <link rel="icon" sizes="48x48" href="/img/global/icons/favicon.ico" >
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```
Comment (lazorfuzz) [inline src/index.html]: We don't need to import this stylesheet, we already already have the `@material-ui/icons` package installed.

File: src/js/components/Ballot/OfficeItemCompressed.jsx [hunk 2]
```diff
@@ -506,14 +506,19 @@ export default class OfficeItemCompressed extends Component {
...
                   {' '}
                   candidates
+                  <i className="material-icons ios-arrow">arrow_forward_ios</i>
                 </span>
-              ) :
-                <span>show more</span>
+              ) : (
+                <span>
+                  show more
+                  <i className="material-icons ios-arrow">arrow_forward_ios</i>
```
Comment (lazorfuzz) [inline src/js/components/Ballot/OfficeItemCompressed.jsx]: Use `<ArrowForwardIcon />`

File: src/index.html (hunk 1 above)
Comment (uroshima) [reply src/index.html]: Good to know, I will exclude the stylesheet, thanks @lazorfuzz

File: src/js/components/Ballot/OfficeItemCompressed.jsx (hunk 2 above)
Comment (uroshima) [reply src/js/components/Ballot/OfficeItemCompressed.jsx]: Cool, thanks @lazorfuzz

**Comment by uroshima** [comment]
I will create a new pull request and delete this one

**Pull request closed without merging**